### PR TITLE
Fixes DF-2349 - updates deputy director to Meredith Fuchs

### PR DESCRIPTION
**Pull request by anselmbradford**
Updates deputy director to Meredith Fuchs.

## Additions

- Added Meredith Fuchs to Leadership calendar filter.

## Changes

- Updated Deputy Director information to Meredith Fuchs.
- Updated `/about-rich-cordray/` URL to `/about-director/`.
- Updated `/about-meredith-fuchs/` URL to `/about-deputy-director/`.
- Normalized director and deputy director photos to be format `NAME-WxH.jpg`.
- Fixed gulp copy task that was missing copying PDFs in subdirectories.
- Fixed `figure` element that was inside a `p`.

## Testing

- Should be Meredith Fuchs on `/the-bureau/`.
- Should be Meredith Fuchs on `/the-bureau/about-deputy-director/`.
- New URL `/the-bureau/about-director/`
- Visit `/newsroom/press-resources/`
- Visit `/about-us/`

## Review

- @sebworks 
- @jimmynotjim 
- @duelj 

## Screenshots

screen shot 2015-08-17 at 1 40 59 pm

screen shot 2015-08-17 at 1 48 19 pm

screen shot 2015-08-17 at 1 48 50 pm

screen shot 2015-08-17 at 1 49 25 pm

## Todos

- There are some slight differences in image sizes because I used our existing media image sizing classes. The round images with accompanying link list should be turned into a macro that allows the same code to generate the media image blocks on `/newsroom/press-resources/`, `/the-bureau/`, `/the-bureau/about-deputy-director/`, and `/the-bureau/about-director/`. These currently have slightly different behaviors across responsive sizes. See [GHE]/flapjack/Modules-V1/issues/5
- The `float-media` and `rwd-crop` classes should be incorporated into the media image structure in `media.less`.

### CODE REVIEW

File: src/about-us/index.html [hunk 1]
```diff
@@ -47,30 +47,32 @@
...
+                                    media_image__150"
+                             src="/static/img/deputy-director-fuchs-round-300x300.jpg"
+                             alt="Meredith Fuchs">
                     </div>
                     <div class="media_body">
                         <span class="h6">Deputy Director</span>
                         <h2 class="bureau-bio_name">Meredith Fuchs</h2>
-                        {# TODO: Add Fuchs's bio #}
-                        <a class="jump-link u-link-disabled">
+                        <a class="jump-link jump-link__underline"
```
Comment (KimberlyMunoz) [inline src/about-us/index.html]: Thank you for catching the Meredith-related todos!

Comment (anselmbradford) [inline src/about-us/index.html]: :smile:

File: src/the-bureau/about-deputy-director/index.html [hunk 2]
```diff
@@ -16,28 +16,24 @@
...
         </p>
         <p>
-            Mr. Antonakes began his professional career as an entry level bank examiner with the Commonwealth of Massachusetts Division of Banks in 1990. He later served in numerous managerial capacities before being appointed by successive Governors to serve as the Commissioner of Banks from December 2003 until November 2010, becoming only the second career bank examiner to ever serve in that capacity.
+            Prior to joining the CFPB, she served as Chief Investigative Counsel of the United States House of Representatives Committee on Energy and Commerce. Previously, Ms. Fuchs held positions as Vice President and General Counsel of the National Security Archive at George Washington University, a litigation partner in private practice, the Supreme Court Assistance Project Fellow at the Public Citizen Litigation Group, and an officer on the D.C. Bar Board of Governors. She is the recipient of the American Library Association’s James Madison Award. Ms. Fuchs served as a law clerk for Judge Patricia M. Wald on the D.C. Circuit Court of Appeals and Judge Paul L. Friedman on the United States District Court for the District of Columbia.
```
Comment (KimberlyMunoz) [inline src/the-bureau/about-deputy-director/index.html]: Semantic line breaks? It makes it easier to catch diffs with content fixes.

**Comment by KimberlyMunoz** [comment]
Looks good to me. Thanks for going through and doing some light fixes with our headings while adding this biography.

File: src/_includes/macros/filters.html [hunk 3]
```diff
@@ -230,6 +230,9 @@
                     {{ _filter_checkbox('calendar', 'Richard Cordray')|safe }}
                 </label>
                 <label class="form-group_item">
+                    {{ _filter_checkbox('calendar', 'Meredith Fuchs')|safe }}
```
Comment (jimmynotjim) [inline src/_includes/macros/filters.html]: Should there be spaces around the pipe here and elsewhere? I saw a similar change in the Refresh PR (have we standardized this yet?)

File: src/the-bureau/about-deputy-director/index.html (hunk 2 above)
Comment (jimmynotjim) [inline src/the-bureau/about-deputy-director/index.html]: There's a few spots that could use line breaks

File: src/_includes/macros/filters.html (hunk 3 above)
Comment (anselmbradford) [inline src/_includes/macros/filters.html]: It hasn't been standardized yet (relevant https://github.com/cfpb/front-end/issues/75). It seems like the trend is toward a space? I've been adding it for new code, but in this case `filters.html` has so many pipes throughout the file that it seemed out of scope. Should I update them here?

File: src/the-bureau/about-deputy-director/index.html (hunk 2 above)
Comment (anselmbradford) [inline src/the-bureau/about-deputy-director/index.html]: Added line breaks in https://github.com/cfpb/cfgov-refresh/commit/ecc5165e5d81e995ff8d7ceb024c6c32d17ea5a6

**Comment by jimmynotjim** [comment]
:+1:

**Pull request merged**